.github/workflows/pull_request.yml

```diff
@@ -29,7 +29,6 @@ jobs:
           python-version: '3.x'
 
       - name: create ci config
-        shell: bash -l {0}
         run: |
           pip install git+https://github.com/obspy/obspy_github_api@${OBSPY_GITHUB_BRANCH}
```

Comment:
Want to change this to `python -m pip` for consistency?

Reply:
Done